### PR TITLE
loginIncorrectInput  Login with array of incorrect data

### PR DESCRIPTION
**Pull request by kalianchuk**
login(role) refactor

### CODE REVIEW

File: modules/user/_page/LoginPage.js [hunk 1]
```diff
@@ -21,16 +21,14 @@ class LoginPage extends AppPage {
...
   }
 
-  get invalidEmailNotification(){
-    return $('//div[@class="invalid-feedback"]')
+  get invalidEmailNotification() {
+    return $('//div[@class="invalid-feedback"]');
   }
 
   login(role) {
-    this.open();
```
Comment (st-newyork) [inline modules/user/_page/LoginPage.js:29]: Why do you remove these lines?

Comment (st-newyork) [reply modules/user/_page/LoginPage.js:29]: Login process won't work without them

Comment (kalianchuk) [reply modules/user/_page/LoginPage.js:29]: It's my bad, I'll roll it back

Comment (st-newyork) [reply modules/user/_page/LoginPage.js:29]: Pause should be back too, at least for now

File: modules/user/_page/LoginPage.js [hunk 2]
```diff
@@ -21,16 +21,15 @@ class LoginPage extends AppPage {
...
+  get invalidEmailNotification() {
+    return $('//div[@class="invalid-feedback"]');
   }
 
   login(role) {
     this.open();
     this.emailInput.setValue(role.email);
     this.passwordInput.setValue(role.password);
     this.submitBtn.click();
-    browser.pause(2000);
```
Comment (st-newyork) [inline modules/user/_page/LoginPage.js:33]: Bring pause back)

**Pull request closed without merging**